src/main/java/com/android/tools/build/bundletool/io/ApkSerializerManager.java

```diff
@@ -191,7 +198,12 @@ public void populateApkSetBuilder(
                         (ModuleSplit split) -> {
                           ZipPath apkPath = apkPathManager.getApkPath(split);
                           return executorService.submit(
-                              () -> apkSerializer.serialize(split, apkPath));
+                              () -> {
+                                ImmutableList<ApkDescription> list =
+                                    apkSerializer.serialize(split, apkPath);
+                                logVerboseInfo(split, apkPath);
```

Comment:
Sorry, I realized that I suggested this location, but actually at the end of the  .serialize() method is probably better.